src/lib/addons/webhook.ts

```diff
@@ -39,9 +47,9 @@ export default class Webhook extends Addon {
             try {
                 extraHeaders = JSON.parse(customHeaders);
             } catch (e) {
-                this.logger.warn(
-                    `Could not parse the json in the customHeaders parameter. [${customHeaders}]`,
-                );
+                state = 'successWithErrors';
+                stateDetails =
+                    'Could not parse the JSON in the customHeaders parameter.';
```

Comment:
Didn't include the `customHeaders` in the details because of potential sensitive information being included.